src/core/task/index.ts

```diff
@@ -3490,6 +3525,17 @@ export class Task {
 			details += terminalDetails
 		}
 
+		// Add recently modified files section
+		const recentlyModifiedFiles = this.fileContextTracker.getAndClearRecentlyModifiedFiles()
+		console.log("recently modified files: ", recentlyModifiedFiles)
+		if (recentlyModifiedFiles.length > 0) {
+			details +=
+				"\n\n# Recently Modified Files\nThese files have been modified since Cline last accessed them (file was just edited so you may need to re-read it before editing):"
```

Comment:
"since Cline last access them" should be changed to "since you last accessed them"